tools/test/h5repack/CMakeTests.cmake

```diff
@@ -527,6 +527,9 @@ macro (ADD_H5_TEST testname)
     set_tests_properties (H5REPACK-${ctest_testname} PROPERTIES
         DEPENDS H5REPACK-${ctest_testname}-clear-objects
     )
+
+    set (ARG_CLEANUP_DEPENDS "H5REPACK-${ctest_testname}")
```

Comment:
It looks like this could be overwritten below by successive calls like `set (ARG_CLEANUP_DEPENDS "H5REPACK-${ctest_testname}_DFF")`. I'm guessing you probably want a top-level set call followed only by `list (APPEND ...)` calls